datalake/backend/tests/test_lambda_functions.py

```diff
@@ -3,10 +3,13 @@
 """
 
 import json
+import os
 import uuid
 
 from pytest import mark
 
+ENV = os.environ["DEPLOY_ENV"]
```

Comment:
We should be able to pull this environment variable out in only one place, and import it elsewhere.